themes/default/content/docs/guides/adopting/from_kubernetes.md

```diff
@@ -356,32 +356,31 @@ Resources:
 
 <img src="/logos/tech/helm.svg" align="right" class="h-32 px-8 pb-4">
 
-Pulumi supports deploying Helm charts as-is without modification, including both Helm V2 and Helm V3. Pulumi renders the templates and applies them directly, much like with `ConfigFile` and `ConfigGroup` shown earlier, which means all provisioning happens client-side using your Kubernetes authentication setup without needing a server-side component such as Tiller.
+Pulumi supports two distinct means of using Helm Charts:
+
+1. [Emulating Helm Charts to render resource templates](#emulating-helm-charts-with-chart-resources)
+2. [Natively installing Helm Charts as Releases](#natively-installing-helm-charts-as-releases)
+
+We discuss and provide examples for each approach in this section.
+
+### Emulating Helm Charts With Chart Resources
+
+With the [Helm V2]({{< relref "/registry/packages/kubernetes/api-docs/helm/v2/release" >}}) and [Helm V3]({{< relref "/registry/packages/kubernetes/api-docs/helm/v3/release" >}}) chart resources, Pulumi renders the templates and applies them directly, much like with `ConfigFile` and `ConfigGroup` shown earlier, which means all provisioning happens client-side using your Kubernetes authentication setup without needing a server-side component such as Tiller (for Helm V2).
```

Comment:
I wonder if we should drop mentions of Helm v2 here while we're at it

Reply:
Done.